src/main/java/com/continuum/nova/NovaRenderer.java

```diff
@@ -68,7 +70,7 @@ public NovaRenderer() {
     public void onResourceManagerReload(@Nonnull IResourceManager resourceManager) {
         this.resourceManager = resourceManager;
 
-        if(firstLoad) {
+        if (firstLoad) {
```

Comment:
My code style is weird. I used to use the Vertical For Loop

```for( int i = 0;
    i < x;
    i++ );```

Then I moved on to super-spaced out everything

```void foo( int x );```

But I've since abandoned both of those.

I like not having spaces before parens in method calls and declarations and for/if/while statements is that this way it feels like the things inside the parens are more connected to the function/if/for/whatever. I recognize that this isn't what many people do, though, so I won't enforce this on your code. I booed on this line because it was my code, but had been reformatted. I'd rather we not reformat each other's code (and if I reformat yours, please call me out on it)